### PR TITLE
added: support rhel8

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

**Comment by blattms** [comment]
I am surely lacking the skills to review this, but I am curious.

Does this change mean that we drop rhel6 (as we drop devtools needed for it) and now just support the newer versions?

**Comment by akva2** [comment]
No, the %{?!el8 means include this when el8 is not defined, without the ! it is included if el8 is defined.

File: redhat/opm-simulators.spec [hunk 1]
```diff
@@ -114,14 +116,15 @@ Requires:       ptscotch-mpich
 %description mpich-bin
 This package contains the applications for opm-simulators
 
+%global debug_package %{nil}
+
 %prep
 %setup -q -n %{name}-release-%{version}-%{tag}
 
 %build
-scl enable devtoolset-6 bash
```
Comment (akva2) [inline redhat/opm-simulators.spec]: or was this what you were referring to markus? this line is void and a leftover after prior attempts. each line in the spec executes in a separate shell so it has no effect. this is why i have to pass the compilers manually below.

**Comment by blattms** [comment]
thanks for the clarification.

**Pull request merged**